static/app/views/dashboardsV2/utils.tsx

```diff
@@ -158,6 +163,16 @@ export function getWidgetInterval(
   const desiredPeriod = parsePeriodToHours(interval);
   const selectedRange = getDiffInMinutes(datetimeObj);
 
+  if (widget.widgetType === WidgetType.METRICS) {
```

Comment:
should we update ` WidgetType.METRICS` to `WidgetType.RELEASES` in a follow-up PR?